openreview/conference/templates/withdraw_process.py

```diff
@@ -1,4 +1,4 @@
-def process(client, note, invitation):
+def process(client, note, invitation, existing_note):
```

Comment:
don't you need to change the signature to `process_update` here? so the process function is being called when the note is edited or deleted.

Reply:
I did that but it's failing with `InvitationExpiredError` now.